src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  margin: 0;
+  padding: 0;
+  background-color: #cecece;
+}
+
+header {
+  width: 100%;
+  box-sizing: border-box;
+  background-color: rgb(255, 255, 255);
+  max-height: 60px;
```

Comment:
No need for this line, it is said in the task to set height of 60px only to links